client/components/first-view/index.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PureRenderMixin from 'react-pure-render/mixin';
+import ReactCSSTransitionGroup from 'react-addons-css-transition-group';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import RootChild from 'components/root-child';
+import { getSectionName } from 'state/ui/selectors';
+import { shouldViewBeVisible } from 'state/first-view/selectors';
+import * as FirstViewActions from 'state/first-view/actions';
```

Comment:
Minor, but thought I'd mention it's good to get into the habit of importing explicitly, so we can reap the reduced chunk size benefits of either the babel tree shaking plugin, or webpack2.